lib/output_helper.rb

```diff
@@ -161,7 +161,7 @@ def self.find_type_and_complete_id(activity)
     def self.build_csv_activity(name, route, activity)
       type, complete_id = find_type_and_complete_id(activity)
       [
-        route && route[:vehicle_id],
+        route && route[:original_vehicle_id],
```

Comment:
@fab-girard asked if you can add a comment like the following so that we can go back when mt-web supports the first version you implemented 
`# TODO: Can be reverted when mt-web supports importing multiple plans with multiple route references`